cmd/osbuild-composer/composer.go

```diff
@@ -104,10 +105,31 @@ func (c *Composer) InitWeldr(repoPaths []string, weldrListener net.Listener) err
 		return fmt.Errorf("Error loading repositories for %s: %v", hostDistro.Name(), err)
 	}
 
+	// Load provider-specific repositories based on available image types
```

Comment:
Make this generic, based on the image name so that it doesn't need to do gcp specific tests and name mangling.